### ISSUE TITLE
Admin is blank after login

### ISSUE THREAD

**Issue by morbidick** [proudcommerce/docker-oxid6#17]
After checking the project out and running `docker-compose up` I get an error screen and I'm unable to do anything further (i only changed the loglevel in the config)
![image](https://user-images.githubusercontent.com/381975/93723277-90714100-fb9d-11ea-9a92-72ca3ecf3e5b.png)

Complete log output:
```
[2020-09-21 00:02:36] OXID Logger.ERROR: EXCEPTION_SYSTEMCOMPONENT_TEMPLATENOTFOUND login.tpl ["[object] (OxidEsales\\Eshop\\Core\\Exception\\SystemComponentException(code: 0): EXCEPTION_SYSTEMCOMPONENT_TEMPLATENOTFOUND login.tpl at /var/www/html/vendor/oxid-esales/oxideshop-ce/source/Core/UtilsObject.php:232)\n[stacktrace]\n#0 /var/www/html/source/oxfunctions.php(104): OxidEsales\\EshopCommunity\\Core\\UtilsObject->oxNew('OxidEsales\\\\Esho...')\n#1 /var/www/html/vendor/oxid-esales/oxideshop-ce/source/Core/ShopControl.php(481): oxNew('OxidEsales\\\\Esho...')\n#2 /var/www/html/vendor/oxid-esales/oxideshop-ce/source/Core/ShopControl.php(359): OxidEsales\\EshopCommunity\\Core\\ShopControl->_render(Object(OxidEsales\\Eshop\\Application\\Controller\\Admin\\LoginController))\n#3 /var/www/html/vendor/oxid-esales/oxideshop-ce/source/Core/ShopControl.php(282): OxidEsales\\EshopCommunity\\Core\\ShopControl->formOutput(Object(OxidEsales\\Eshop\\Application\\Controller\\Admin\\LoginController))\n#4 /var/www/html/vendor/oxid-esales/oxideshop-ce/source/Core/ShopControl.php(142): OxidEsales\\EshopCommunity\\Core\\ShopControl->_process('OxidEsales\\\\Esho...', NULL, NULL, NULL)\n#5 /var/www/html/vendor/oxid-esales/oxideshop-ce/source/Core/Oxid.php(27): OxidEsales\\EshopCommunity\\Core\\ShopControl->start()\n#6 /var/www/html/source/index.php(16): OxidEsales\\EshopCommunity\\Core\\Oxid::run()\n#7 {main}\n"] []
[2020-09-21 00:02:36] OXID Logger.ERROR: Function 'getIsOrderStep' does not exist or is not accessible! (OxidEsales\Eshop\Application\Controller\Admin\LoginController)
 ["[object] (OxidEsales\\Eshop\\Core\\Exception\\SystemComponentException(code: 0): Function 'getIsOrderStep' does not exist or is not accessible! (OxidEsales\\Eshop\\Application\\Controller\\Admin\\LoginController)\n at /var/www/html/vendor/oxid-esales/oxideshop-ce/source/Core/Base.php:84)\n[stacktrace]\n#0 /var/www/html/vendor/oxid-esales/oxideshop-ce/source/Core/ViewConfig.php(842): OxidEsales\\EshopCommunity\\Core\\Base->__call('getIsOrderStep', Array)\n#1 /var/www/html/source/tmp/smarty/c5138306ad2b64a1c736b9aee924cfc0^%%81^810^810F9158%%account_menu.tpl.php(32): OxidEsales\\EshopCommunity\\Core\\ViewConfig->getShowCompareList()\n#2 /var/www/html/vendor/smarty/smarty/libs/Smarty.class.php(1876): include('/var/www/html/s...')\n#3 /var/www/html/source/tmp/smarty/c5138306ad2b64a1c736b9aee924cfc0^%%02^02C^02CD7FDB%%exception.tpl.php(30): Smarty->_smarty_include(Array)\n#4 /var/www/html/vendor/smarty/smarty/libs/Smarty.class.php(1270): include('/var/www/html/s...')\n#5 /var/www/html/vendor/oxid-esales/oxideshop-ce/source/Internal/Framework/Smarty/Legacy/LegacySmartyEngine.php(66): Smarty->fetch('message/excepti...', 'login')\n#6 /var/www/html/vendor/oxid-esales/oxideshop-ce/source/Internal/Framework/Templating/TemplateRenderer.php(35): OxidEsales\\EshopCommunity\\Internal\\Framework\\Smarty\\Legacy\\LegacySmartyEngine->render('message/excepti...', Array)\n#7 /var/www/html/vendor/oxid-esales/oxideshop-ce/source/Core/ShopControl.php(509): OxidEsales\\EshopCommunity\\Internal\\Framework\\Templating\\TemplateRenderer->renderTemplate('message/excepti...', Array)\n#8 /var/www/html/vendor/oxid-esales/oxideshop-ce/source/Core/ShopControl.php(359): OxidEsales\\EshopCommunity\\Core\\ShopControl->_render(Object(OxidEsales\\Eshop\\Application\\Controller\\Admin\\LoginController))\n#9 /var/www/html/vendor/oxid-esales/oxideshop-ce/source/Core/ShopControl.php(282): OxidEsales\\EshopCommunity\\Core\\ShopControl->formOutput(Object(OxidEsales\\Eshop\\Application\\Controller\\Admin\\LoginController))\n#10 /var/www/html/vendor/oxid-esales/oxideshop-ce/source/Core/ShopControl.php(142): OxidEsales\\EshopCommunity\\Core\\ShopControl->_process('OxidEsales\\\\Esho...', NULL, NULL, NULL)\n#11 /var/www/html/vendor/oxid-esales/oxideshop-ce/source/Core/Oxid.php(27): OxidEsales\\EshopCommunity\\Core\\ShopControl->start()\n#12 /var/www/html/source/index.php(16): OxidEsales\\EshopCommunity\\Core\\Oxid::run()\n#13 {main}\n"] []
[2020-09-21 00:02:36] OXID Logger.ERROR: Function 'getComponent' does not exist or is not accessible! (OxidEsales\Eshop\Application\Controller\Admin\LoginController)
 ["[object] (OxidEsales\\Eshop\\Core\\Exception\\SystemComponentException(code: 0): Function 'getComponent' does not exist or is not accessible! (OxidEsales\\Eshop\\Application\\Controller\\Admin\\LoginController)\n at /var/www/html/vendor/oxid-esales/oxideshop-ce/source/Core/Base.php:84)\n[stacktrace]\n#0 /var/www/html/vendor/oxid-esales/oxideshop-ce/source/Application/Component/Widget/WidgetController.php(45): OxidEsales\\EshopCommunity\\Core\\Base->__call('getComponent', Array)\n#1 /var/www/html/vendor/oxid-esales/oxideshop-ce/source/Core/ShopControl.php(398): OxidEsales\\EshopCommunity\\Application\\Component\\Widget\\WidgetController->init()\n#2 /var/www/html/vendor/oxid-esales/oxideshop-ce/source/Core/WidgetControl.php(131): OxidEsales\\EshopCommunity\\Core\\ShopControl->_initializeViewObject('OxidEsales\\\\Esho...', NULL, Array, NULL)\n#3 /var/www/html/vendor/oxid-esales/oxideshop-ce/source/Core/ShopControl.php(278): OxidEsales\\EshopCommunity\\Core\\WidgetControl->_initializeViewObject('OxidEsales\\\\Esho...', NULL, Array, Array)\n#4 /var/www/html/vendor/oxid-esales/oxideshop-ce/source/Core/ShopControl.php(142): OxidEsales\\EshopCommunity\\Core\\ShopControl->_process('OxidEsales\\\\Esho...', NULL, Array, Array)\n#5 /var/www/html/vendor/oxid-esales/oxideshop-ce/source/Core/WidgetControl.php(65): OxidEsales\\EshopCommunity\\Core\\ShopControl->start('oxwlanguagelist', NULL, Array, Array)\n#6 /var/www/html/vendor/oxid-esales/oxideshop-ce/source/Core/Smarty/Plugin/function.oxid_include_widget.php(32): OxidEsales\\EshopCommunity\\Core\\WidgetControl->start('oxwlanguagelist', NULL, Array, Array)\n#7 /var/www/html/source/tmp/smarty/c5138306ad2b64a1c736b9aee924cfc0^%%93^93D^93D6031B%%header.tpl.php(39): smarty_function_oxid_include_widget(Array, Object(Smarty))\n#8 /var/www/html/vendor/smarty/smarty/libs/Smarty.class.php(1876): include('/var/www/html/s...')\n#9 /var/www/html/source/tmp/smarty/c5138306ad2b64a1c736b9aee924cfc0^%%36^366^366ECF91%%page.tpl.php(14): Smarty->_smarty_include(Array)\n#10 /var/www/html/vendor/smarty/smarty/libs/Smarty.class.php(1876): include('/var/www/html/s...')\n#11 /var/www/html/source/tmp/smarty/c5138306ad2b64a1c736b9aee924cfc0^%%02^02C^02CD7FDB%%exception.tpl.php(36): Smarty->_smarty_include(Array)\n#12 /var/www/html/vendor/smarty/smarty/libs/Smarty.class.php(1270): include('/var/www/html/s...')\n#13 /var/www/html/vendor/oxid-esales/oxideshop-ce/source/Internal/Framework/Smarty/Legacy/LegacySmartyEngine.php(66): Smarty->fetch('message/excepti...', 'ox|0|0|1|0|ssl')\n#14 /var/www/html/vendor/oxid-esales/oxideshop-ce/source/Internal/Framework/Templating/TemplateRenderer.php(35): OxidEsales\\EshopCommunity\\Internal\\Framework\\Smarty\\Legacy\\LegacySmartyEngine->render('message/excepti...', Array)\n#15 /var/www/html/vendor/oxid-esales/oxideshop-ce/source/Core/ShopControl.php(509): OxidEsales\\EshopCommunity\\Internal\\Framework\\Templating\\TemplateRenderer->renderTemplate('message/excepti...', Array)\n#16 /var/www/html/vendor/oxid-esales/oxideshop-ce/source/Core/ShopControl.php(359): OxidEsales\\EshopCommunity\\Core\\ShopControl->_render(Object(OxidEsales\\Eshop\\Application\\Controller\\ExceptionErrorController))\n#17 /var/www/html/vendor/oxid-esales/oxideshop-ce/source/Core/ShopControl.php(282): OxidEsales\\EshopCommunity\\Core\\ShopControl->formOutput(Object(OxidEsales\\Eshop\\Application\\Controller\\ExceptionErrorController))\n#18 /var/www/html/vendor/oxid-esales/oxideshop-ce/source/Core/ShopControl.php(758): OxidEsales\\EshopCommunity\\Core\\ShopControl->_process('exceptionError', 'displayExceptio...')\n#19 /var/www/html/vendor/oxid-esales/oxideshop-ce/source/Core/ShopControl.php(144): OxidEsales\\EshopCommunity\\Core\\ShopControl->_handleSystemException(Object(OxidEsales\\Eshop\\Core\\Exception\\SystemComponentException))\n#20 /var/www/html/vendor/oxid-esales/oxideshop-ce/source/Core/Oxid.php(27): OxidEsales\\EshopCommunity\\Core\\ShopControl->start()\n#21 /var/www/html/source/index.php(16): OxidEsales\\EshopCommunity\\Core\\Oxid::run()\n#22 {main}\n"] []
[2020-09-21 00:02:36] OXID Logger.ERROR: OxidEsales\Eshop\Application\Controller\ExceptionErrorController is not an instance of OxidEsales\Eshop\Application\Component\Widget\WidgetController ["[object] (OxidEsales\\Eshop\\Core\\Exception\\ObjectException(code: 0): OxidEsales\\Eshop\\Application\\Controller\\ExceptionErrorController is not an instance of OxidEsales\\Eshop\\Application\\Component\\Widget\\WidgetController at /var/www/html/vendor/oxid-esales/oxideshop-ce/source/Core/UtilsObject.php:232)\n[stacktrace]\n#0 /var/www/html/source/oxfunctions.php(104): OxidEsales\\EshopCommunity\\Core\\UtilsObject->oxNew('OxidEsales\\\\Esho...', 'OxidEsales\\\\Esho...')\n#1 /var/www/html/vendor/oxid-esales/oxideshop-ce/source/Core/WidgetControl.php(135): oxNew('OxidEsales\\\\Esho...', 'OxidEsales\\\\Esho...')\n#2 /var/www/html/vendor/oxid-esales/oxideshop-ce/source/Core/ShopControl.php(278): OxidEsales\\EshopCommunity\\Core\\WidgetControl->_initializeViewObject('exceptionError', 'displayExceptio...', NULL, NULL)\n#3 /var/www/html/vendor/oxid-esales/oxideshop-ce/source/Core/ShopControl.php(758): OxidEsales\\EshopCommunity\\Core\\ShopControl->_process('exceptionError', 'displayExceptio...')\n#4 /var/www/html/vendor/oxid-esales/oxideshop-ce/source/Core/ShopControl.php(144): OxidEsales\\EshopCommunity\\Core\\ShopControl->_handleSystemException(Object(OxidEsales\\Eshop\\Core\\Exception\\SystemComponentException))\n#5 /var/www/html/vendor/oxid-esales/oxideshop-ce/source/Core/WidgetControl.php(65): OxidEsales\\EshopCommunity\\Core\\ShopControl->start('oxwlanguagelist', NULL, Array, Array)\n#6 /var/www/html/vendor/oxid-esales/oxideshop-ce/source/Core/Smarty/Plugin/function.oxid_include_widget.php(32): OxidEsales\\EshopCommunity\\Core\\WidgetControl->start('oxwlanguagelist', NULL, Array, Array)\n#7 /var/www/html/source/tmp/smarty/c5138306ad2b64a1c736b9aee924cfc0^%%93^93D^93D6031B%%header.tpl.php(39): smarty_function_oxid_include_widget(Array, Object(Smarty))\n#8 /var/www/html/vendor/smarty/smarty/libs/Smarty.class.php(1876): include('/var/www/html/s...')\n#9 /var/www/html/source/tmp/smarty/c5138306ad2b64a1c736b9aee924cfc0^%%36^366^366ECF91%%page.tpl.php(14): Smarty->_smarty_include(Array)\n#10 /var/www/html/vendor/smarty/smarty/libs/Smarty.class.php(1876): include('/var/www/html/s...')\n#11 /var/www/html/source/tmp/smarty/c5138306ad2b64a1c736b9aee924cfc0^%%02^02C^02CD7FDB%%exception.tpl.php(36): Smarty->_smarty_include(Array)\n#12 /var/www/html/vendor/smarty/smarty/libs/Smarty.class.php(1270): include('/var/www/html/s...')\n#13 /var/www/html/vendor/oxid-esales/oxideshop-ce/source/Internal/Framework/Smarty/Legacy/LegacySmartyEngine.php(66): Smarty->fetch('message/excepti...', 'ox|0|0|1|0|ssl')\n#14 /var/www/html/vendor/oxid-esales/oxideshop-ce/source/Internal/Framework/Templating/TemplateRenderer.php(35): OxidEsales\\EshopCommunity\\Internal\\Framework\\Smarty\\Legacy\\LegacySmartyEngine->render('message/excepti...', Array)\n#15 /var/www/html/vendor/oxid-esales/oxideshop-ce/source/Core/ShopControl.php(509): OxidEsales\\EshopCommunity\\Internal\\Framework\\Templating\\TemplateRenderer->renderTemplate('message/excepti...', Array)\n#16 /var/www/html/vendor/oxid-esales/oxideshop-ce/source/Core/ShopControl.php(359): OxidEsales\\EshopCommunity\\Core\\ShopControl->_render(Object(OxidEsales\\Eshop\\Application\\Controller\\ExceptionErrorController))\n#17 /var/www/html/vendor/oxid-esales/oxideshop-ce/source/Core/ShopControl.php(282): OxidEsales\\EshopCommunity\\Core\\ShopControl->formOutput(Object(OxidEsales\\Eshop\\Application\\Controller\\ExceptionErrorController))\n#18 /var/www/html/vendor/oxid-esales/oxideshop-ce/source/Core/ShopControl.php(758): OxidEsales\\EshopCommunity\\Core\\ShopControl->_process('exceptionError', 'displayExceptio...')\n#19 /var/www/html/vendor/oxid-esales/oxideshop-ce/source/Core/ShopControl.php(144): OxidEsales\\EshopCommunity\\Core\\ShopControl->_handleSystemException(Object(OxidEsales\\Eshop\\Core\\Exception\\SystemComponentException))\n#20 /var/www/html/vendor/oxid-esales/oxideshop-ce/source/Core/Oxid.php(27): OxidEsales\\EshopCommunity\\Core\\ShopControl->start()\n#21 /var/www/html/source/index.php(16): OxidEsales\\EshopCommunity\\Core\\Oxid::run()\n#22 {main}\n"] []
[21 Sep 00:02:36.423939 2020] [uncaught error] [type E_ERROR] [file /var/www/html/vendor/oxid-esales/oxideshop-ce/source/Core/UtilsObject.php] [line 232] [code ] [message Uncaught OxidEsales\Eshop\Core\Exception\ObjectException: OxidEsales\Eshop\Application\Controller\ExceptionErrorController is not an instance of OxidEsales\Eshop\Application\Component\Widget\WidgetController in /var/www/html/vendor/oxid-esales/oxideshop-ce/source/Core/UtilsObject.php:232
Stack trace:
#0 /var/www/html/source/oxfunctions.php(104): OxidEsales\EshopCommunity\Core\UtilsObject->oxNew('OxidEsales\\Esho...', 'OxidEsales\\Esho...')
#1 /var/www/html/vendor/oxid-esales/oxideshop-ce/source/Core/WidgetControl.php(135): oxNew('OxidEsales\\Esho...', 'OxidEsales\\Esho...')
#2 /var/www/html/vendor/oxid-esales/oxideshop-ce/source/Core/ShopControl.php(278): OxidEsales\EshopCommunity\Core\WidgetControl->_initializeViewObject('exceptionError', 'displayExceptio...', NULL, NULL)
#3 /var/www/html/vendor/oxid-esales/oxideshop-ce/source/Core/ShopControl.php(758): OxidEsales\EshopCommunity\Core\ShopControl->_process('exceptionError', 'displayExceptio...')
#4 /var/www/html/vendor/oxid-esales/oxideshop-ce/source/Core/Sho]
```

Do you have any hints how i could debug this any further?

**Comment by tabsl** [comment]
- database ist fully installed?
- frontend and backend are not working / white screen?
- anywhere an redirect to the start page?

**Comment by morbidick** [comment]
Hi @tabsl ,
* the database seems fully installed, is there a test command i could run?
* frontend is working fine, i can even login as a normal user. Only `/admin` is behaving wierd
* i can't see any redirects, clean urls is working

**Comment by tabsl** [comment]
ah, did you check the error message?! ;-) seems like admin templates are missing ...

`EXCEPTION_SYSTEMCOMPONENT_TEMPLATENOTFOUND login.tpl`

**Comment by morbidick** [comment]
Yeah I saw that but can't explain why. Since this project is bootstrapping a new Oxid Installation when its started for the first time, is this a problem with the composer package itself?

**Comment by tabsl** [comment]
I also don´t know. Have you tried a fully new install?

**Comment by morbidick** [comment]
oh damn, i had an old version without the fix from #16. @tabsl Thank you so much for your time!
